### PR TITLE
Demonstrate moment locales

### PR DESCRIPTION
**Pull request by andrew-nowak**
**Overall change:** Creates a storybook page demonstrating how moment translations look, as will be displayed in simorgh (ie with psammead-locales added on top)

**Code changes:**

- Takes heavy inspiration (copy+pastes) from psammead-locales storybook pages.

---

- [ ] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

File: src/app/lib/utilities/moment/index.stories.jsx [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+// be exact
+// https://momentjs.com/docs/#/customization/relative-time-threshold/
+moment.relativeTimeThreshold('s', 60);
+moment.relativeTimeThreshold('ss', 3);
+moment.relativeTimeThreshold('m', 60);
+moment.relativeTimeThreshold('h', 24);
+moment.relativeTimeThreshold('d', 30);
+moment.relativeTimeThreshold('M', 12);
+
+const THIS_DATE = 946826177000;
```
Comment (andrew-nowak) [inline src/app/lib/utilities/moment/index.stories.jsx]:
```suggestion
// Sunday, 2 January 2000 15:16:17 GMT+00:00
const A_DATE = 946826177000;
```

**Review by ghost — COMMENTED** [review]
Why do we have both `5 x` and `10 x`? i.e `5 hours` and `10 hours`.

**Comment by andrew-nowak** [comment]
@sadickisaac good point, was a premature optimisation for Russian (or other languages with multiple levels of pluralisation) which I believe isn't in the first four phases so can be left out for now

**Review by ghost — APPROVED** [review]

**Review by DenisHdz — APPROVED** [review]
Very nice 💅🏼

**Comment by david-boydell** [comment]
Looks good. I'm not sure what we need to do re CodeClimate but once that's sorted we are ready for merge.

**Comment by andrew-nowak** [comment]
The codeclimate issues in this PR are about code duplication which this team has agreed to ignore in test/storybook code for the sake of keeping code explicit.

(side note, looking at the code changes in this PR, I find it kinda funny that it has only detected two issues of duplication 😄 )

**Pull request merged**